Gemfile

```diff
@@ -17,6 +17,9 @@ end
 
 gem "cheffish", ">= 17"
 
+# Explicitly avoid 1.x that doesn't support Ruby 2.4+
+gem "json", ">= 2.0.0"
```

Comment:
Isn't this a bundled gem in ruby itself now?

Reply:
This could be my limited understanding of gem/ruby world, but here is why I added it just to get public comments
- Looks like json was added in ruby 2.4+ and I wasn't sure which min version does chef require to run
- I checked the [rails](https://github.com/rails/rails/blob/54707cb7a71fa3677cab35c139df6ca3ad685566/Gemfile#L32) project and they have added this as some sort of best practice? so I re-used the same here.
Please suggest the correct direction.

Reply:
We require Ruby 2.7+ so it should be built in